src/screens/Components.tsx

```diff
@@ -13,6 +14,11 @@ const Home = ({ navigation }: Props) => {
   return (
     <View style={styles.container}>
       <Text>Components screen</Text>
+      <Input
```

Comment:
Jeśli to jest input tekstowy to fajnie jakbyś go jednak nazwał `TextInput`, bo nazwa w tym momencie sugeruje, że jest on generyczny i mógłby być również typu `number`, `password` itp.

W tym przypadku, gdy importujesz `TextInput` z react-native, to możesz zrobić coś takiego żeby uniknąć duplikacji zmiennych:
`import { TextInput as ReactTextInput } from 'react-native'`